AccountService/src/UserInputService.java

```diff
@@ -1,24 +1,81 @@
+import java.util.ArrayList;
 import java.util.Scanner;
 
 public class UserInputService {
-    Scanner loginInput = new Scanner(System.in);
-    LoginService loginService;
-
-    public UserInputService(LoginService loginService) {
-        this.loginService = loginService;
-    }
-    public void loginInput() {
-        System.out.println("Enter username");
-        String userName = loginInput.nextLine();
-        System.out.println("Enter password");
-        String password = loginInput.nextLine();
-
-        if (loginService.verifyUser(userName, password)) {
-            System.out.println("Username and password is valid");
-        }
-        else {
-            System.out.println("Username and password is invalid");
-
-        }
-    }
+	Scanner userInput = new Scanner(System.in);
+	LoginService loginService;
+	CustomerService customerService;
+
+	public UserInputService(LoginService loginService, CustomerService customerService) {
+		this.loginService = loginService;
+		this.customerService = customerService;
+	}
+
+	public void loginInput() {
+		System.out.println("Enter username");
+		String userName = userInput.nextLine();
+		System.out.println("Enter password");
+		String password = userInput.nextLine();
+
+		if (loginService.verifyUser(userName, password)) {
+			System.out.println("Username and password is valid");
+		} else {
+			System.out.println("Username and password is invalid");
+
+		}
+	}
+
+	public void createAccountInput() {
+		ArrayList<User> newUserDatabase = customerService.userList;
+		customerService.createData();
```

Comment:
I'm thinking we shouldn't add `customerService.createData()` here, since it does not fit within this method objective. It might be more suitable for this line to be in Main.java when customerService object is initalized. I'm open for debate though?